modules/AOS_Quotes/language/en_us.lang.php

```diff
@@ -201,5 +201,12 @@
     'LBL_SHIPPING_TAX_AMT_USDOLLAR' => 'Shipping Tax (Default Currency)',
     'LBL_GRAND_TOTAL_USDOLLAR' => 'Grand Total (Default Currency)',
     'LBL_QUOTE_TO' => 'Quote To',
+
+    'LBL_SUBTOTAL_TAX_AMOUNT_USDOLLAR' => 'Subtotal Tax (Default Currency)',
```

Comment:
is this a subtotal `of` tax only or `Subtotal + Tax` ?
In modules/AOS_Line_Item_Groups/language/en_us.lang.php the string is:
`'LBL_SUBTOTAL_TAX_AMOUNT_USDOLLAR' => 'Subtotal + Tax (Default Currency)',`